### PR TITLE
Add schema and create database

### PR DESCRIPTION
**Pull request by njgingrich**
Save the SQL commands to create a sqlite database to hold all the taskmaster information, as well as the (empty) database.

### CODE REVIEW

File: src/schema/schema.sql [hunk 1]
```diff
@@ -0,0 +1,65 @@
+CREATE TABLE `seasons` (
+  `id` integer UNIQUE PRIMARY KEY,
+  `name` text NOT NULL,
+  `type` text NOT NULL -- 'regular or champion of champions'
```
Comment (cbrochtrup) [inline src/schema/schema.sql:4]: Good idea to add a type here.

File: src/schema/schema.sql [hunk 2]
```diff
@@ -0,0 +1,65 @@
+CREATE TABLE `seasons` (
```
Comment (cbrochtrup) [inline src/schema/schema.sql]: What do you think about calling this `series` to be consistent with the [taskmaster nomenclature](https://taskmaster.fandom.com/wiki/Series_1)?

**Review by cbrochtrup — APPROVED** [review]
Looks nice. Thanks for adding this!

File: src/schema/schema.sql (hunk 2 above)
Comment (njgingrich) [reply src/schema/schema.sql]: That's a good idea, I'll update.

**Pull request merged**